.spin/bin/env

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# Advice on how to get the path of the current script is endless - this seems
+# to work for what we want on the systems we want
+SCRIPT_PATH=`dirname "$0"`
+SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
```

Comment:
We can use [command substitution](https://tldp.org/LDP/abs/html/commandsub.html#CSPARENS) syntax here which is easier to nest than quotes:

```suggestion
SCRIPT_PATH=$( dirname "$0" )
SCRIPT_PATH=$( cd "$SCRIPT_PATH" && pwd )
```

Reply:
Agreed, I would avoid spaces inside `$()` though for consistency with all other Bash scripts in the repo.

Reply:
Agreed. Edited my suggestion to reflect that.